lib/logr.rb

```diff
@@ -0,0 +1,45 @@
+require "logger"
+
+require "logr/version"
+require "logr/entry"
+require "logr/json_formatter"
+
+module Logr
+  class Logger
+    def initialize(name, level=::Logger::DEBUG, log_device=STDOUT)
+      @logger = ::Logger.new(log_device).tap do |logger|
+        logger.formatter = JSONFormatter.new
+        logger.progname = name
+        logger.level = level
+      end
+    end
+
+    def event(name, options={})
+      Entry.new(@logger).event(name, options)
+    end
+
+    def metric(name, value, type: "counter")
+      Entry.new(@logger).metric(name, value, type: type)
```

Comment:
`Metric` should verify that `value` is countable/measureable and not allow `String` for instance.